src/helpers/emailTemplates.js

```diff
@@ -0,0 +1,48 @@
+import dotenv from 'dotenv';
+import Token from './tokenHelper';
+
+dotenv.config();
+const { EMAIL_ADDRESS } = process.env;
+
+/**
+ * @export
+ * @class EmailService
+ */
+class EmailTemplates {
```

Comment:
Remember to rename the files of everywhere you have used class to PASCALCASE, for instance, EmailTemplates as in the class name is